Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Undo/PrefabUndoBase.h

```diff
@@ -36,6 +36,7 @@ namespace AzToolsFramework
             void virtual Redo(InstanceOptionalConstReference instanceToExclude);
 
         protected:
+            bool m_changed;
```

Comment:
It's a good practice to keep datatypes of the small size like bools at the end of all member variables. This ensures that these small types can take advantage of padding and reduce the memory allocation for the objects of the class.